### PR TITLE
Replace Quick Start content with Kilrcoda exercise

### PR DESCRIPTION
**Pull request by StackScribe**
Signed-off-by: Meg McRoberts <meg.mcroberts@dynatrace.com>

This PR replaces the old Quick Start contents with a link to the kilrcoda tutorial exercise followed by a few links to get the users started with their own Keptn cluster.

### CODE REVIEW

**Review by agardnerIT — CHANGES_REQUESTED** [review]
A huge improvement (to me). Few comments:

- Usage of "Keptn's": In less than an hour, you will learn the main featres of Keptn and how to implement them.

- Please re-include the Keptn CLI section

- I agree that we shouldn't mention k3d as it is opinionated and moves us to a situation of "where do we stop"? To use Keptn, a user should provide a k8s cluster. How they do that is up to them.

**Comment by StackScribe** [comment]
Thanks, Adam!  I think I cut in your comments.  I modified the titles of a couple sections that are linked, and modified the intro text for the Install section.  More work is required on all of these but we can at least have reasonable titles for the links in the Quick Start

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
As we discussed before, Kilrcoda is not ready to be the only option until we get all the questions sorted. IMHO other options should be kept here or moved to other pages

**Comment by StackScribe** [comment]
This does not touch the existing tutorials.  It merely replaces the QuickStart, which doesn't work and has almost no content.  Do you want to leave the QuickStart in place and just add this under a different title?

File: content/docs/0.16.x/manage/service/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Service
+title: Deploying Services
```
Comment (oleg-nenashev) [inline content/docs/0.16.x/manage/service/index.md:2]: Unrelated

File: content/docs/0.16.x/operate/advanced_install_options/index.md [hunk 2]
```diff
@@ -1,11 +1,9 @@
 ---
-title: Advanced Install Options
+title: Install Keptn using the Helm chart
```
[the remark this answers is not in the document]
Comment (StackScribe) [reply content/docs/0.16.x/operate/advanced_install_options/index.md:2]: Again, replacing this title is in the plans and, since this text links to that file, I want to replace the title.  This one seems particularly critical since we are on the brink of making the Helm chart the only installation option so we do not want new users thinking of this as an "advanced option".

[the remark this answers is not in the document]
Comment (StackScribe) [reply content/docs/0.16.x/operate/advanced_install_options/index.md:2]: You said to restore the old Quick Start without modifications so that is what I did.  Do you instead want me to add a brief statement to the Quick Start that this gives instructions for installing Keptn on a local k8s cluster and running some exercises, and link to the new stuff as a way to explore Keptn without having to install anything locally?

**Comment by StackScribe** [comment]
You said to restore the old Quick Start without modifications so that is what I did. Do you instead want me to add a brief statement to the Quick Start that this gives instructions for installing Keptn on a local k8s cluster and running some exercises, and link to the new stuff as a way to explore Keptn without having to install anything locally?

**Comment by StackScribe** [comment]
> So now Quickstart does not reference Killrcoda at all. Also unrelated changes
You said to restore the old Quick Start without modifications so that is what I did. Do you instead want me to add a brief statement to the Quick Start that this gives instructions for installing Keptn on a local k8s cluster and running some exercises, and link to the new stuff as a way to explore Keptn without having to install anything locally?

File: content/docs/0.16.x/manage/service/index.md (hunk 1 above)
Comment (oleg-nenashev) [reply content/docs/0.16.x/manage/service/index.md:2]: Bonus points for using separate pull requests next time. Or at least mentioning  such refactorings in the PR title

**Review by oleg-nenashev — APPROVED** [review]
LGTM though a lot of unrelated changes are confusing

**Comment by oleg-nenashev** [comment]
> You said to restore the old Quick Start without modifications so that is what I did

This is not what I said. What I said is that the old tutorials should be retained. There are already two tutorials path on the quickstart so that you could have added a third one. This is what we discussed a number of times already

**Comment by agardnerIT** [comment]
> "Kilrcoda is not ready to be the only option until we get all the questions sorted".

What questions are outstanding?

"There are already two tutorials path on the quickstart so that you could have added a third one"

Have they been updated to work with new Keptn versions (0.15.1 +)? AFAIK the only tutorial that works with 0.15.1 + is the killercoda tutorial.
